cmd/register/register.go

```diff
@@ -210,9 +210,15 @@ func readIPMIDetails(eth0Mac string, hal hal.InBand) (*models.ModelsV1MachineIPM
 		if bmc == nil {
 			return nil, errors.New("unable to read ipmi bmc info configuration")
 		}
+
+		err := hal.BMCChangePassword(hal.BMCSuperUser(), "pw") //TODO pw should be provided via cmdline, for example
```

Comment:
So the password must then be provided by a kernel commandline ?

Reply:
Or we introduce a grpc endpoint for this